views/confirm-company.html

```diff
@@ -0,0 +1,91 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Confirm the company
+{% endblock %}
+
+{% block backLink %}
+  {{ govukBackLink({
+    text: "Back",
+    href: ""
```

Comment:
you can set the back link to the company number screen, we know it'll be the previous screen